docs/tutorials/ifci_log_qgroundcontrol.rst

```diff
@@ -0,0 +1,95 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _ifci_log:
+
+********************************************
+Exporting IFCI Logs Using QGroundControl
+********************************************
+
+IFCI data can be found in the flight controller logs. These logs can be exported from the flight controller using QGroundControl.
+
+Click the QGroundControl icon on the upper left-hand corner of the application.
+Then click "Analyze Tools" from the pop-up menu.
+
+.. figure:: ../_static/tutorial_images/ifci_log/analyze_tools.png
+    :align: center
+    :alt: Analyze Tools
+
+    QGroundControl Analyze Tools
+
+
+Initially, there may be no logs displayed. If so, click the "Refresh" button to display the logs available on your flight controller.
+
+.. figure:: ../_static/tutorial_images/ifci_log/refresh_button.png
+    :align: center
+    :alt: Refresh Button
+
+    QGroundControl Refresh Logs
+
+
+Then select the log that you want to download, and click the "Download" button. 
+
+.. figure:: ../_static/tutorial_images/ifci_log/download_button.png
+    :align: center
+    :alt: Download Button
+
+    QGroundControl Download Logs
+
+
+You can then choose where to download the selected log file through the file explorer. The file will be downloaded as a .ulg file.
```

Comment:
Again, if it's PX4 specific, then .ulg is all that will come up, but if this tutorial includes ardupilot then we should also expect a different log type (I think they're .bin maybe?).